docs/streams/index.html

```diff
@@ -18,10 +18,24 @@
 <script><!--#include virtual="../js/templateData.js" --></script>
 
 <script id="streams-template" type="text/x-handlebars-template">
-    <h1>Kafka Streams API</h1>
+    <h1>Kafka Streams</h1>
```

Comment:
Any changes to the AK Streams landing page must be coordinated with @derrickdoo.  We already have changes planned here, so we shouldn't push anything out unless is coordinated/reviewed appropriately.